x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/action/SetOperationModeAction.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+package org.elasticsearch.xpack.core.indexlifecycle.action;
+
+import org.elasticsearch.action.Action;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.support.master.AcknowledgedRequest;
+import org.elasticsearch.action.support.master.AcknowledgedResponse;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+import org.elasticsearch.common.xcontent.ToXContentObject;
+import org.elasticsearch.xpack.core.indexlifecycle.OperationMode;
+
+import java.io.IOException;
+import java.util.Objects;
+
+public class SetOperationModeAction extends Action<SetOperationModeAction.Request, SetOperationModeAction.Response> {
+    public static final SetOperationModeAction INSTANCE = new SetOperationModeAction();
+    public static final String NAME = "cluster:admin/xpack/index_lifecycle/operation_mode";
```

Comment:
I think this should be: `cluster:admin/xpack/index_lifecycle/operation_mode/set`